pkg/discovery/summary_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright the Sonobuoy contributors 2022
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package discovery
+
+import (
+	"testing"
+        "encoding/json"
+	"bytes"
+	"flag"
+        "path/filepath"
+	"io/ioutil"
+)
+
+const (
+	//Root directory where all the file sin the tarball are located
+	//Simulates the directory where all the files resulting from discovery end up before being compressed into the tarball
+	//For the test, this directory is the location where the test files are located
+	tarballRootDir = "testdata"
+
+	//Name of the golden file that stores the expected result
+	SummaryGoldenFileName = "summary_test.golden"
```

Comment:
These constants are just test specific; just define them in the test. Thats just the norm for tests otherwise you explode with constants. I know its kind of the opposite of the other case in a normal method but in normal code those strings/constants are pretty rare but every single test case usually has 1 or more constants.